source/release-notes.txt

```diff
@@ -10,6 +10,25 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.8.0 Changelog
+---------------
+
+*Released July 09, 2024*
+
+New features:
+
+- New conditional mapping rules to add advanced feature support for:
+  - Polymorphic relational schemas, where different rows in the same table 
+    relate to entities that need to be treated differently in the MongoDB model
+  - Relational models with foreign keys that imply “one to many” relationships 
+    but where in reality they are one-to-one and should be represented as 
+    embedded documents, not arrays
```

Comment:
(simple sentences)
This is a bit of a run on sentence. Can we simplify it a little?